pybossa/cache/announcements.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf8 -*-
+# This file is part of PYBOSSA.
+#
+# Copyright (C) 2015 Scifabric LTD.
```

Comment:
this can be updated to 2021.

Reply:
Fixed via https://github.com/bloomberg/pybossa/commit/08230b6aeb6ff0b5ec32066729dd36e7e0c88cab